server_plan9.go

```diff
@@ -0,0 +1,30 @@
+//go:build plan9
+// +build plan9
```

Comment:
These build tags should be unnecessary since the filename already ends with `_plan9.go`.